exegol/model/ExegolImage.py

```diff
@@ -577,7 +584,7 @@ def getDownloadSize(self) -> str:
 
     def getSize(self) -> str:
         """Image size getter. If the image is installed, return the on-disk size, otherwise return the remote size"""
-        return self.__disk_size if self.__is_install else f"[bright_black]{self.__dl_size} (compressed)[/bright_black]"
+        return self.__disk_size if self.__is_install else f"[bright_black]~{self.__remote_est_size}[/bright_black]"
```

Comment:
You have a duplicate here, you can now call getRealSize() instead of having the same `if`